lib/data/repository/recovery_words_repository_impl.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:fluffychat/data/datasource/recovery_words_data_source.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/model/recovery_words/recovery_words.dart';
+import 'package:fluffychat/domain/repository/recovery_words_repository.dart';
+
+class RecoveryWordsRepositoryImpl implements RecoveryWordsRepository {
+  final RecoveryWordsDataSource recoveryWordsDataSource = getIt.get<RecoveryWordsDataSource>();
```

Comment:
use shorter version: `final recoveryWordsDataSource = getIt.get<RecoveryWordsDataSource>();`

Reply:
It's not shorter version it's unsafer version 😂